travis/samples/ngx-h2mem-social-ws-nol2/.yo-rc.json

```diff
@@ -28,7 +28,7 @@
       "protractor"
     ],
     "serverPort": "8080",
-    "jhiPrefix": "jhi",
+    "jhiPrefix": "myPrefix",
```

Comment:
You are faster than my thoughts :) i was just asking you about this in another issue

Reply:
But the build didn't failed!!!
The build is green: https://travis-ci.org/jhipster/generator-jhipster/jobs/318478008
Look the log, during the `$JHIPSTER_SCRIPTS/02-generate-project.sh` :

```
ERROR in [at-loader] ./src/test/javascript/spec/test.module.ts:10:75 
    TS2305: Module '"/home/travis/app/src/main/webapp/app/shared/index"' has no exported member 'JhiTrackerService'.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
If you find JHipster useful consider supporting our collective https://opencollective.com/generator-jhipster

Server application generated successfully.

Run your Spring Boot application:
 ./mvnw

Client application generated successfully.

Start your Webpack development server with:
 yarn start
```

I think I found the problem:
https://github.com/jhipster/generator-jhipster/blob/master/generators/client/index.js#L396

Maybe it's because we didn't handle the error with `(err) =>` for this `spawnCommandSync`

Reply:
I won't be able to test this change until tonight, as my day work is starting now...